packages/frontend-2/components/server-management/DeleteUserDialog.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
Would be better to handle the actual GQL mutation from this component, otherwise if you want to use this dialog elsewhere you'll have to duplicate the mutation logic each time from the parent component.